Documentation/ApiOverview/Authentication/Index.rst

```diff
@@ -199,14 +199,14 @@ a form of final transformation on the login data.
 For "authUserFE" and "authUserBE" subtypes, the :php:`authUser()` method may
 return different values:
 
-- a negative value indicates that the authentication has definitely failed
+- a negative value or 0 indicates that the authentication has definitely failed
   and that no other "auth" service should be called up.
 
 - a positive value smaller than 100 indicates that the authentication
```

Comment:
Here we should explicitly mention the > 0 (or >= 1) requirement

Reply:
?

Reply:
Instead of `positive value` we should write `value larger than 0` since we already have a confusing logic here.